geopandas/geoseries.py

```diff
@@ -566,3 +566,7 @@ def __sub__(self, other):
             stacklevel=2,
         )
         return self.difference(other)
+
+    def __covers__(self, other):
+        """Implement covers operator as for builtin set type"""
+        return self.covers(other)
```

Comment:
This shouldn't be needed (since there is no operator in Python that can be used for this)